docker-compose-rag-doll-only.yml

```diff
@@ -0,0 +1,70 @@
+services:
+  slackbot:
+    build: slackbot
+    ports:
+      - ${SLACK_BOT_PORT}:${SLACK_BOT_PORT}
+    environment:
+      RABBITMQ_DEFAULT_USER: ${RABBITMQ_DEFAULT_USER}
+      RABBITMQ_DEFAULT_PASS: ${RABBITMQ_DEFAULT_PASS}
+      RABBITMQ_QUEUE_USER_CHATS: ${RABBITMQ_QUEUE_USER_CHATS}
+      RABBITMQ_QUEUE_USER_CHAT_REPLIES: ${RABBITMQ_QUEUE_USER_CHAT_REPLIES}
+      RABBITMQ_HOST: ${RABBITMQ_HOST}
+      RABBITMQ_PORT: ${RABBITMQ_PORT}
+      SLACK_BOT_PORT: ${SLACK_BOT_PORT}
+      SLACK_BOT_TOKEN: ${SLACK_BOT_TOKEN}
+      SLACK_SIGNING_SECRET: ${SLACK_SIGNING_SECRET}
+    depends_on:
+      - rabbitmq
+    restart: unless-stopped
```

Comment:
could you please remove all the restart behavior

Reply:
Done.